cypress/e2e/manager_login.cy.js

```diff
@@ -0,0 +1,28 @@
+import testUser from "../../credentials/managerCredentials.json"
+import LoginPage from "../pages/LoginPage.js"
+
+describe('tests for login function', () => {
+  beforeEach(() => {
+    cy.visit('/Account/Login?ReturnUrl=%2F')
+  })
+  it('Negative scenario - try to login without credentials', () => {
+    LoginPage.loginButton().click()
+    LoginPage.emptyLoginFieldMessageCheck()
+  })
+
+  it('Negative scenario - login with incorrect credentials', () => {
+    LoginPage.submitLogin(testUser.incorrectLogin,testUser.incorrectPass)
+    LoginPage.incorrectLoginFieldMessageCheck()
```

Comment:
lepiej to nazwac: verifyIncorrectLoginMsg

Reply:
ok, nazwy zmienione